tests/drivers/flash/common/testcase.yaml

```diff
@@ -31,13 +31,15 @@ tests:
       - nrf52840dk/nrf52840
   drivers.flash.common.default:
     filter: ((CONFIG_FLASH_HAS_DRIVER_ENABLED and not CONFIG_TRUSTED_EXECUTION_NONSECURE)
-      and dt_label_with_parent_compat_enabled("storage_partition", "fixed-partitions"))
+      and (dt_label_with_parent_compat_enabled("storage_partition", "fixed-partitions")
+      or dt_label_with_parent_compat_enabled("storage_partition", "nordic,owned-partitions")))
     integration_platforms:
       - qemu_x86
       - mimxrt1060_evk
   drivers.flash.common.no_explicit_erase:
```

Comment:
@de-nordic I see that Twister on nrf54l15pdk executes both _drivers.flash.common.default_ and _drivers.flash.common.no_explicit_erase_
However, I don't see any difference in test configuration (Kconfig / overlay) between these two.
Is this intentional?

Reply:
Long time in the original code there was, because it was possible to configure the nrf54l15 driver to have and to not have explicit-erase requirement. It is no longer a case.